src/cluster.c

```diff
@@ -753,7 +753,7 @@ static int store_replica_nodes(dict *nodes, dict *replicas) {
  * Only parse primary nodes if the `parsed_primary_id` argument is NULL,
  * otherwise replicas are also parsed and its primary_id is returned by pointer
  * via 'parsed_primary_id'. */
-static int parse_cluster_nodes_line(valkeyClusterContext *cc, char *line,
+static int parse_cluster_nodes_line(valkeyClusterContext *cc, valkeyContext *c, char *line,
```

Comment:
Mention what `c` is in the comment above.

The return value VALKEY_OK/ERR isn't mentioned either.

Reply:
Updated comment looks good.